src/Commands/core/ArchiveDumpCommands.php

```diff
@@ -588,4 +593,71 @@ private function destinationCleanup(string $destination): string
         }
         return $destination;
     }
+
+    /**
+     * Converts symlinks to full content for an archive.
+     *
+     * @param $convert_symlinks
+     *  Whether to convert symlinks.
+     * @param $archivePath
+     *   The file path of the archive.
+     *
+     * @return void
+     */
+    public function convertSymlinks(
```

Comment:
Please add param types and return types. Remove any duplicate info in the Doxygen.